src/Service/Helper/HelpMessageHelper.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\acquia_lift\Service\Helper\HelpMessageHelper.
+ */
+
+namespace Drupal\acquia_lift\Service\Helper;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Utility\LinkGeneratorInterface;
+use Drupal\Core\Url;
+
+class HelpMessageHelper {
+  /**
+   * Acquia Lift credential settings.
+   *
+   * @var array
+   */
+  private $credentialSettings;
+
+  /**
+   * Link generator.
+   *
+   * @var \Drupal\Core\Utility\LinkGeneratorInterface
+   */
+  private $linkGenerator;
+
+  /**
+   * Constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory service.
+   * @param \Drupal\Core\Utility\LinkGeneratorInterface $link_generator
+   *   The link generator.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, LinkGeneratorInterface $link_generator) {
+    $this->credentialSettings = $config_factory->get('acquia_lift.settings')->get('credential');
+    $this->linkGenerator = $link_generator;
+  }
+
+  /**
+   * Get help message (by route name).
+   *
+   * @param string $route_name
+   *   Route name.
+   *
+   * @return string
+   *   The help message.
+   */
+  public function getMessage($route_name) {
+    switch ($route_name) {
```

Comment:
Should probably include the overview help here as well.  See https://api.drupal.org/api/drupal/core%21modules%21help%21help.api.php/function/hook_help/8 for more information about the route provided to match on as well a link to the standard help template.